views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
```

Comment:
I have a little trouble with this one. Here's what I have figured out how
to do:

    <p>With whom:
        {{event.attending.length}} attending so far
          <li><class="event" id=eventDetail>
          {% for people in event.attending %}
              <ul>{{people}}</ul>
        {% endfor %}</li>
      </p>

The remaining problem is that only the first name shows up with a bullet
point. The second name is now moved to its own line (that's good!) but it
doesn't look like it's coding as a list item because it doesn't have a
bullet point. Still working on this; suggestions appreciated.

On Mon, Feb 9, 2015 at 10:23 PM, David Hatch <notifications@github.com>
wrote:

> In views/event-detail.html
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25-23discussion-5Fr24386722&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=iEEbZJsjXUypN4l1RPo5C4SzYo6dzfWhKAs_xC_47Gg&s=wf-WCYb1r1IEK8mE0I6ImK6UyIui-kj2mAFkNhi1jto&e=>
> :
>
> > @@ -0,0 +1,34 @@
> > +{% extends "layout.html" %}
> > +
> > +{% block mainContent %}
> > +  <div>
> > +    <p>What: {{event.title}}</p>
> > +    <p>When: {{event.date}}</p>
> > +    <p>Where: {{event.location}}</p>
> > +    <p>With whom: {{event.attending}}</p>
>
> This should be enumerated using the for template helper, and embedded
> within a ul so that each person attending appears as a li.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25_files-23r24386722&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=iEEbZJsjXUypN4l1RPo5C4SzYo6dzfWhKAs_xC_47Gg&s=6-BQ_IKeHob77MHWsHo5339e_y6DDGkXqMk4WO4rv50&e=>
> .
>